{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,81 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly. Feel free to remove any
+sections that won't be relevant to reviewers of your PR, including this
+one.
+
+Anything marked with `TODO:` is something that you, the developer, should
+do as part of the PR.
+
+The PR submission process is described in [more detail here.](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/index.html)
+
+### Testing
+- TODO: Include unit, integration, acceptance tests as appropriate
+- TODO: Include accessibility (a11y) tests
+- TODO: Include tests that capture the external-query scaling properties
+- [ ] Check that Database migrations are backwards-compatible
+- [ ] Manually test right-to-left languages and i18n
+  of the changes.
+
+### DevOps assistance
+If it's needed, or even if you suspect it might be needed, do any/all these
```

Comment:
I'd also expect to see this content in the CONTRIBUTING document rather than the template for the actual PR message.